plugins/metadata/plugin.go

```diff
@@ -119,6 +171,10 @@ func init() {
 				return nil, err
 			}
 
+			config := *bolt.DefaultOptions
```

Comment:
Do we need to call `defaultBoltOptions` here instead of using the default options provided by bolt (as we overwrote `NoFreelistSync` at https://github.com/containerd/containerd/pull/10745/files#diff-9c09b2ff8e763fbd2736030258f48ef2115b4cf95d00dc1e32871cedde707fc0R80-R84

Reply:
We could start with an empty struct too. I guess it doesn't make too much difference here since we overwrite the majority of values via plugin's configuration.